src/ledger.rs

```diff
@@ -90,7 +91,7 @@ pub fn next_entries_mut(
         let mut entries = Vec::with_capacity(transactions.len() / chunk_len + 1);
 
         for chunk in transactions.chunks(chunk_len) {
```

Comment:
Unrelated to this PR, I'm confused why this is `transactions` and not `transactions[chunk_len..]`.